src/modules/app/route-configs/EventsRouteConfig.js

```diff
@@ -52,16 +54,20 @@ class EventsRouteConfig implements RouteConfig<EventsRouteParamsType, RequiredPa
     return this.getRoutePath({ city, language })
   }
 
-  getRequiredPayloads = (payloads: AllPayloadsType): RequiredPayloadsType => ({ events: payloads.eventsPayload })
+  getRequiredPayloads = (payloads: AllPayloadsType): RequiredPayloadsType => ({
+    cities: payloads.citiesPayload,
+    events: payloads.eventsPayload
+  })
 
-  getPageTitle = ({ t, payloads, cityName, location }) => {
-    if (!cityName) {
+  getPageTitle = ({ t, payloads, location }) => {
```

Comment:
The cityName isn't passed to this function? What about the other routes? Why does it work for e.g. `CategoriesRoute`?

Reply:
Oh yes, it is, but as I need the complete cityModel anyways, I decided not to take it from there